operator/hack/olm-operator-install.sh

```diff
@@ -15,35 +15,28 @@ function main() {
     ;;
   esac
 
-  local -r operator_ns="${1:-}"
-  local -r image_tag_base="${2:-}"
 
   case $# in
-  3)
-    local -r index_version="${3:-}"
-    local -r operator_version="${3:-}"
-    ;;
-  4)
-    local -r index_version="${3:-}"
-    local -r operator_version="${4:-}"
+  5)
+    local -r operator_ns="${1:-}"
+    local -r index_image_repo="${2:-}"
+    local -r index_image_tag="${3:-}"
+    local -r starting_csv_version="${4:-}"
+    local -r operator_channel="${5:-}"
```

Comment:
[nit, just noticed] If I read this correctly, "${6:-}" makes an argument optional but the following usage and the script proceeds as if the arguments are required.

I think this should be:
```suggestion
    local -r operator_ns="${1}"
    local -r index_image_repo="${2}"
    local -r index_image_tag="${3}"
    local -r starting_csv_version="${4}"
    local -r operator_channel="${5}"
```